packages/cli/commands/init.js

```diff
@@ -107,8 +109,11 @@ exports.handler = async options => {
     auth: authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
     c,
     account: optionalAccount,
+    useHiddenConfig,
   } = options;
-  const configPath = (c && path.join(getCwd(), c)) || getConfigPath();
+  const configPath =
+    (c && path.join(getCwd(), c)) ||
```

Comment:
It might make this easier to read if we use the `config` variable directly rather than the alias `c`.

Reply:
Actually I take that back. I wonder why we didn't just call it `configPath` 🤔. You can ignore this initial comment.

Reply:
Going to resolve for now, feel free to reopen if necessary